code/game/machinery/telecomms/machine_interactions.dm

```diff
@@ -254,8 +254,12 @@
 		var/mob/living/silicon/ai/U = user
 		multitool = U.aiMulti
 	else if(iscyborg(user) && in_range(user, src))
-		if(istype(user.get_active_held_item(), /obj/item/multitool))
-			multitool = user.get_active_held_item()
+		var/mob/living/silicon/robot/borguser = user
+		if(borguser.model.toolbox)
```

Comment:
Is it correct here to check the entire toolbox and not just see if we have it activated in the tool hand?

Reply:
We could check the hand, I just wasn't sure it was meaningfully worth the extra loop.

Reply:
Maybe I am misunderstanding here but why would you need a loop?
You just want to check if the active held item is the omnitool, then if the active tool is a multitool, no?
Maybe this proc is not called in the circumstances I expect it to be

Reply:
Honestly my point was a nothingburger, it's been changed.

Reply:
I still don't quite understand why you are using a loop here instead of just checking the _active tool_, presumably this would let you activate the multitool behaviour of some items without having the multitool selected?
If you can explain that to me then I think we're good